static/templates/user_profile_modal.handlebars

```diff
@@ -20,16 +20,22 @@
         </div>
         <hr class="user-profile-modal-hr">
         {{#if profile_data}}
-        {{/if}}
```

Comment:
I added 5f10fa1692392dc0f59f86da358ea3b6d04bdb52 to resolve this change more correctly; the actual issue here was that this whole conditional was useless.